config/samples/first-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  replicas: {{ .Params.Replicas }} # tells deployment to run 2 pods matching the template
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+        - name: nginx
+          image: nginx:1.7.9
+          env:
+            - name: ENVIRONMENT_VARIABLE
+              value: "{{ .Params.Param }}"
```

Comment:
this isn't covered in the material... we should add it in the docs or remove it from the same

Reply:
I wrote it in description. This is not final draft, I want to move iteratively not write 10 pages of docs at once. This will come in later PRs. This PR does not close the whole documentation issue.